controllers/topology/internal/mergepatch/mergepatch.go

```diff
@@ -37,11 +37,11 @@ type mergePatchHelper struct {
 	patch []byte
 }
 
-// newMergePatchHelper will return a patch that yields the modified document when applied to the original document.
+// New will return a patch that yields the modified document when applied to the original document.
 // NOTE: In the case of ClusterTopologyReconciler, original is the current object, modified is the desired object, and
 // the patch returns all the changes required to align current to what is defined in desired; fields not defined in desired
 // are going to be preserved without changes.
-func newMergePatchHelper(original, modified client.Object, c client.Client) (*mergePatchHelper, error) {
+func New(original, modified client.Object, c client.Client) (*helper, error) {
```

Comment:
WDYT about calling this `NewHelper` (same as with our `util/patch` helper)

Reply:
Fixed!